<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEducationWindow</class>
 <widget class="QMainWindow" name="AddEducationWindow">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Education Information</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Add Education Information</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelYear">
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputYear"/>
    </item>
    <item>
     <widget class="QLabel" name="labelInstitute">
      <property name="text">
       <string>Institute/College/School</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputInstitute"/>
    </item>
    <item>
     <widget class="QLabel" name="labelDegree">
      <property name="text">
       <string>Degree/Course</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputDegree"/>
    </item>
    <item>
     <widget class="QLabel" name="labelGrades">
      <property name="text">
       <string>Grades</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputGrades"/>
    </item>
    <item>
     <widget class="QLabel" name="labelLocation">
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputLocation"/>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="columnCount">
       <number>5</number>
      </property>
      <property name="horizontalHeaderLabels">
       <stringlist>
        <string>Year</string>
        <string>Institute</string>
        <string>Degree</string>
        <string>Grades</string>
        <string>Location</string>
       </stringlist>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
